### ISSUE TITLE
Installation and Usage of ENVI MET INX

### ISSUE THREAD

**Issue by HighFlyingFalcon** [AntonelloDN/Envimet-inx#1]
I am facing the below issue. I tried on both versions of SketchUp. Kindly help.

![image](https://user-images.githubusercontent.com/62982000/89399085-e1ba9380-d709-11ea-9fa0-e66ab567b9be.png)
![image](https://user-images.githubusercontent.com/62982000/89399539-889f2f80-d70a-11ea-8d09-df0ac09438eb.png)

**Comment by AntonelloDN** [comment]
Hi @HighFlyingFalcon,

The first screenshot is a warning. It is normal to have it with Sketchup 2020 - I need to update the declaration of its compatibility. It could be a problem with Sketchup 2019. I will investigate it but it does not stop you from using the plugin. Can you please provide your Sketchup version? Please, go to "Help/Info about Sketchup". You should see your version (e.g. Sketchup Pro 2020 20.1.235 64-bit).

The second screenshot, can you replicate these steps? [example steps](https://github.com/AntonelloDN/Envimet-inx/wiki/01.-Create-your-first-model-with-Envimet-INX)
Please, let me know if and where you stop.

-Antonello

**Comment by HighFlyingFalcon** [comment]
Hi Antonello,
Thanks for your response.
I was able to figure out the issue by following your instructions, it wasn't because of the version. I missed setting the location using the envimet location icon. All these while I used to set the location using the icon which is found in the bottom left corner of the SketchUp interface.
I am now able to save the file in INX format and able the open the same in ENVI MET as well.

Thanks for your guidance and support.
